lib/queueClient.ts

```diff
@@ -309,4 +331,44 @@ export class QueueClient extends Client {
   async cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void> {
     return this._context.managementClient!.cancelScheduledMessages(sequenceNumbers);
   }
+
+  /**
+   * Lists the sessions on the ServiceBus Queue.
+   * @param skip The number of sessions to skip
+   * @param top Maximum numer of sessions.
+   * @param lastUpdateTime Filter to include only sessions updated after a given time. Default
+   * value: 3 days ago from the current time.
+   */
+  async listMessageSessions(skip: number, top: number, lastUpdatedTime?: Date): Promise<ListSessionsResponse> {
+    return this._context.managementClient!.listMessageSessions(skip, top, lastUpdatedTime);
+  }
+
+  /**
+   * 
+   * @param options 
+   */
+  async acceptSession(options?: AcceptSessionOptions): Promise<MessageSession> {
+    if (!options) options = {};
+    this._context.isSessionEnabled = true;
+    return MessageSession.create(this._context, options);
+  }
+
+  /**
+   * Receives messages from a session enabled Queue.
+   * @param onSessionMessage The message handler to receive service bus messages from a session
+   * enabled Queue.
+   * @param onError The error handler to receive an error that occurs while receiving messages
+   * from a session enabled Queue.
+   */
+  receiveMessgesFromSessions(
```

Comment:
I am guessing this is analogous to the `receive`?
Can the user call `receiveMessgesFromSessions` without calling `acceptSession`?

Reply:
So user cant control which session they are requesting messages from when using `receiveMessagesFromSessions`?

Reply:
How does one "close" the receiver in this case?

The corresponding `receive` in the QueueClient returns a receiveHandler which is then used to close the receiver.

The only way I can see is to close the queue client itself which seems reasonable, but then begs the question why do we support the "receiveHandler" returned by the `QueueClient.receive`?

Reply:
That makes sense, thanks. 

> If this seems confusing then we can look at creating overloads in TS.... Another option would be to name both the methods differently

The current way of using the options bag works just fine. We just have to ensure that the comments tell the whole story :)



This is exactly why we should have a spec where the the expectations are laid out, instead of every new developer trying to reverse engineer another SDK